content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -112,15 +68,33 @@ The Istio control plane must be installed with virtual machine integration enabl
     $ istioctl install -f "${WORK_DIR}"/vmintegration.yaml
     {{< /text >}}
 
+## Configure the VM namespace
+
+1. Create the namespace that will host the virtual machine:
+
+    {{< text bash >}}
+    $ kubectl create namespace "${VM_NAMESPACE}"
+    {{< /text >}}
+
+1. Create a serviceaccount for the virtual machine:
+
+    {{< text bash >}}
+    $ kubectl create serviceaccount "${SERVICE_ACCOUNT}" -n "${VM_NAMESPACE}"
+    {{< /text >}}
+
 ## Create files to transfer to the virtual machine
 
-1. Make a copy of files to copy to the virtual machine
+1. Create a Kubernetes token. This example sets the token expire time to 1 hour:
 
     {{< text bash >}}
-    $ cp -a "${WORK_DIR}"/"${CLUSTER_NAME}"/ca-cert.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/
-    $ cp -a "${WORK_DIR}"/"${CLUSTER_NAME}"/key.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/
-    $ cp -a "${WORK_DIR}"/"${CLUSTER_NAME}"/root-cert.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/
-    $ cp -a "${WORK_DIR}"/"${CLUSTER_NAME}"/workload-cert-chain.pem "${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"/cert-chain.pem
+    $ tokenexpiretime=3600
+    $ echo '{"kind":"TokenRequest","apiVersion":"authentication.k8s.io/v1","spec":{"audiences":["istio-ca"],"expirationSeconds":'$tokenexpiretime'}}' | kubectl create --raw /api/v1/namespaces/$VM_NAMESPACE/serviceaccounts/$SERVICE_ACCOUNT/token -f - | jq -j '.status.token' > "${WORK_DIR}"/istio-token
+    {{< /text >}}
```

Comment:
Sorry if this is not the right place for the report an issue. I tried to run this but I got back the error below.
```
Error from server (NotFound): the server could not find the requested resource
```

Here is version information.
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.0", GitCommit:"e19964183377d0ec2052d1f1fa930c4d7575bd50", GitTreeState:"clean", BuildDate:"2020-08-26T14:30:33Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.11", GitCommit:"ea5f00d93211b7c80247bf607cfa422ad6fb5347", GitTreeState:"clean", BuildDate:"2020-08-13T15:11:47Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```